src/lib/onboarding/service.ts

```diff
@@ -271,4 +309,37 @@ export class OnBoardingService {
 
     return authToken;
   }
+
+  /// FALLBACK ONLY
+  async submitAccountWithSig(accountWithSig: AccountWithSigToSubmit) {
+    return Database.sequelize.transaction(async transaction => {
+      const { address, email } = accountWithSig;
```

Comment:
This format creates one atomic transaction?

Reply:
yes